.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,59 @@
+name: Continuous Integration
+on:
+  push:
+    tags:
+    - '**'
+
+env:
+  GCLOUD_SERVICE_ACCOUNT_KEYFILE: ${{ secrets.GCLOUD_GH_ACTIONS_AUTH }}
+  GCLOUD_SERVICE_ACCOUNT_NAME: rasa-x-ci-github-actions@rasa-platform.iam.gserviceaccount.com
+
+jobs:
+  release:
+    name: Release new demo
+    runs-on: ubuntu-20.04
+    env:
+      TAG_NAME: ${GITHUB_REF#refs/tags/}
+      SAFE_TAG_NAME: $(echo $TAG_NAME | sed 's/\./-/g')
+      BASE_BRANCH: ${{ github.base_ref }}
+      DOCKER_DEPLOYMENT: pr-${SAFE_TAG_NAME}
+    steps:
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+
+      - uses: google-github-actions/setup-gcloud@daadedc81d5f9d3c06d2c92f49202a3cc2b919ba
+        name: Authenticate with gcloud 🎫
+        with:
+          version: "${{ env.GCLOUD_VERSION }}"
+          service_account_email: ${{ env.GCLOUD_SERVICE_ACCOUNT_NAME }}
+          service_account_key: ${{ secrets.GCLOUD_GH_ACTIONS_AUTH }}
+
+      - name: Authenticate docker 🎫
+        run: |
+          # Set up docker to authenticate via gcloud command-line tool.
+          gcloud auth configure-docker
+
+      - name: Build new image
+        run: |
+          RASA_SDK_VERSION=$(curl https://hub.docker.com/v2/repositories/rasa/rasa-sdk/tags |  jq -r '.results|.[]|.name' | grep -E "^[[:digit:]|\.|rc]+$" | sort -V | tail -n 1)
+          echo "Latest SDK version is ${RASA_SDK_VERSION}"
+          docker build \
+                -t gcr.io/rasa-platform/rasa-x-demo:${DOCKER_DEPLOYMENT} \
```

Comment:
It doesn't make sense, the workflow runs only on push a tag event, so pushing a Docker image with a tag "pr-<tag>" is not necessary.

Reply:
arghhh sorry I thought I had removed all the traces of PR stuff :eyes:

Reply:
oh I see what you mean, I can build it already with `TAG_NAME`

Reply:
done! :D